packages/web-react/src/components/Container/Container.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { applyClassNamePrefix } from '../../utils/classname';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+import { ChildrenProps, StyleProps } from '../../types';
+
+export interface ContainerProps extends ChildrenProps, StyleProps {
+  tag: ElementType;
+}
+
+const defaultProps = {
+  tag: 'div',
```

Comment:
will it ever be anything other than div?

Reply:
Do not know. But it should be also `section` or another react component, maybe.

Reply:
This option I have removed and there is only `div` now.